src/Db/Adapter/SqliteAdapter.php

```diff
@@ -1098,8 +1098,12 @@ protected function calculateNewTableColumns(string $tableName, string|false $col
             }
         }
 
-        $selectColumns = array_filter($selectColumns, 'strlen');
-        $writeColumns = array_filter($writeColumns, 'strlen');
+        $selectColumns = array_filter($selectColumns, function ($value) {
+            return strlen($value) > 0;
+        });
+        $writeColumns = array_filter($writeColumns, function ($value) {
+            return strlen($value) > 0;
+        });
```

Comment:
@ADmad I already know you don't like it due to https://github.com/phpstan/phpstan/issues/2266 😛